modules/Bio/EnsEMBL/Compara/PipeConfig/EBI/CITest/BuildCITestMasterDatabase_conf.pm

```diff
@@ -83,9 +83,11 @@ sub default_options {
 
         'division'      => 'citest',
         'compara_dir'   => $self->check_dir_in_ensembl('ensembl-compara/'),
-        'config_dir'    => $self->check_dir_in_ensembl('ensembl-compara/modules/t/citest_division'),
-        'init_reg_conf' => $self->check_file_in_ensembl('ensembl-compara/modules/t/citest_division/production_init_reg_citest_conf.pl'),
-        'reg_conf_tmpl' => $self->check_file_in_ensembl('ensembl-compara/scripts/pipeline/production_reg_citest_conf_tmpl.pl'),
+        # NOTE: 'config_dir' is already in ENV but the check* method is not called on it
+        #       Since it is essential for this pipeline, redefining it here with the checks
+        'config_dir'    => $self->check_dir_in_ensembl('ensembl-compara/conf/citest'),
```

Comment:
I would replace this line to `'config_dir'    => $self->check_dir_in_ensembl('ensembl-compara/conf/' . $self->o('division')),`